src/index.ts

```diff
@@ -136,7 +136,7 @@ initialize({
 app.use(
   "/docs",
   swaggerUi.serve,
-  swaggerUi.setup(undefined, {
+  swaggerUi.setup(apiDoc, {
```

Comment:
```suggestion
  swaggerUi.setup(undefined, {
```

If this the swagger doc thing a problem for you the ticket should stay open and we can continue looking into it, but with this fix it doesn't work as it should anymore, so I think it hides the problem (and would break the production app).